docs/serving/using-auto-tls.md

```diff
@@ -78,170 +92,213 @@ To enable support for automatic TLS certificate provisioning in Knative:
-         ...
-           autoTLS: Enabled
-         ...
-         ```
-
-         Example:
-
-         ```shell
-         apiVersion: v1
-         kind: ConfigMap
-         metadata:
-           name: config-network
-           namespace: knative-serving
-         data:
-           ...
-           autoTLS: Enabled
-           ...
-           httpProtocol: Redirected
-           ...
-         ```
-
-    1. Ensure that the file was updated successfully:
-
-        ```shell
-        kubectl get configmap config-network --namespace knative-serving --output yaml
-        ```
+1.  Ensure that the file was updated successfully:
+
+    ```shell
+    kubectl get configmap config-certmanager --namespace knative-serving --output yaml
+    ```
+
+### Step 5: Turn on Auto TLS
+
+Update the
+[`config-network` ConfigMap](https://github.com/knative/serving/blob/master/config/config-network.yaml)
+in the `knative-serving` namespace to enable `autoTLS`and specify how HTTP
+requests are handled:
+
+1.  Run the following command to edit your `config-network` ConfigMap:
+
+    ```shell
+    kubectl edit configmap config-network --namespace knative-serving
+    ```
+
+1.  Add the `autoTLS: Enabled` attribute under the `data` section:
+
+    ```shell
+    ...
+    data:
+    ...
+      autoTLS: Enabled
+    ...
+    ```
+
+    Example:
+
+    ```shell
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: config-network
+      namespace: knative-serving
+    data:
+       ...
+       autoTLS: Enabled
+       ...
+    ```
+
+1.  Configure how HTTP and HTTPS requests are handled in the
+  [`httpProtocol`](https://github.com/knative/serving/blob/master/config/config-network.yaml#L110)
+  attribute.
+
+    By default, Knative ingress is configured to serve HTTP traffic
+    (`httpProtocol: Enabled`). Now that your cluster is configured to use TLS
+    certificates and handle HTTPS traffic, you can specify whether or not any
+    HTTP traffic is allowed.
+
+    Supported `httpProtocol` values:
+
+    - `Enabled`: Serve HTTP traffic.
+    - `Disabled`: Rejects all HTTP traffic.
+    - `Redirected`: Responds to HTTP request with a `302` redirect to ask the
+      clients to use HTTPS.
+
+     ```shell
+     ...
+     data:
+     ...
+       autoTLS: Enabled
+     ...
+     ```
+
+     Example:
+
+     ```shell
+     apiVersion: v1
+     kind: ConfigMap
+     metadata:
+       name: config-network
+       namespace: knative-serving
+     data:
+       ...
+       autoTLS: Enabled
+       ...
+       httpProtocol: Redirected
+       ...
+     ```
+
+    **Note:**
+    When using HTTP01 challenge, `httpProtocol` field has to be set to `Enabled` to make sure HTTP01 challenge requests can be accepted by the cluster.
+
+1.  Ensure that the file was updated successfully:
+
+    ```shell
+    kubectl get configmap config-network --namespace knative-serving --output yaml
+    ```
 
 Congratulations! Knative is now configured to obtain and renew TLS certificates.
```

Comment:
Remove "Congratulations!..." - this doesn't really add anything. Instead consider adding the steps to confirm this works as a 'verification' section or similar.